.github/workflows/tests.yml

```diff
@@ -41,8 +41,10 @@ jobs:
           pip uninstall dagrunner -y
 
       # TESTS (inc. test coverage)
+      # excluded logging as per https://github.com/MetOffice/dagrunner/issues/39
       - name: Run pytest + coverage report gen
-        run: pytest --cov=dagrunner --cov-report=term --cov-report=html | tee coverage_output.txt
+        run: pytest --cov=dagrunner --cov-report=term --cov-report=html --ignore=dagrunner/tests/utils/logging/test_integration.py | tee coverage_output.txt; test ${PIPESTATUS[0]} -eq 0
```

Comment:
Exclude logging test in CI until https://github.com/MetOffice/dagrunner/issues/44